core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+class CommandType:
+    MOVE_CALL = 0
+    TRANSFER_OBJECTS = 1
+    SPLIT_COIN = 2
+    MERGE_COINS = 3
+    PUBLISH = 4
+    MAKE_MOVE_VEC = 5
+    UPGRADE = 6
+
+
+class Address:
+    def __init__(self, data):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)